app/views/projects/_status_chooser.html.erb

```diff
@@ -18,7 +18,7 @@
     <div class="col-md-3 col-sm-4 col-xs-4 criteria-radio<%= crypto_class %>">
       <div class="status-chooser">
       <%= f.form_group status_symbol do %>
-        <a href="#<%= criterion.to_s %>">
+        <%= link_to({ anchor: criterion.to_s }) do %>
```

Comment:
That looks better and more idiomatic. I haven't tested it locally but if it behaves as intended, LGTM